bridge/github/import.go

```diff
@@ -42,40 +40,67 @@ func (gi *githubImporter) Init(conf core.Configuration) error {
 
 // ImportAll iterate over all the configured repository issues and ensure the creation of the
 // missing issues / timeline items / edits / label events ...
-func (gi *githubImporter) ImportAll(repo *cache.RepoCache, since time.Time) error {
-	gi.iterator = NewIterator(gi.conf[keyOwner], gi.conf[keyProject], gi.conf[keyToken], since)
+func (gi *githubImporter) ImportAll(ctx context.Context, repo *cache.RepoCache, since time.Time) (<-chan core.ImportResult, error) {
+	gi.iterator = NewIterator(ctx, 10, gi.conf[keyOwner], gi.conf[keyProject], gi.conf[keyToken], since)
+	out := make(chan core.ImportResult)
+	gi.out = out
+
+	go func() {
+		defer close(gi.out)
+
+		// Loop over all matching issues
+		for gi.iterator.NextIssue() {
+			issue := gi.iterator.IssueValue()
+			var b *cache.BugCache
```

Comment:
if the context is cancelled and this variable isn't stil initialised we'll have a `panic` if we try to commit the bug in `select` block. maybe something like this can be better
```golang
if b != nil {
    if err := b.CommitAsNeeded(); err != nil {
	err := errors.Wrap(err, "bug commit")
	out <- core.NewImportError(err, b.Id())
    }
}
```

Reply:
done